x-pack/plugins/siem/public/common/components/exceptions/viewer/index.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPanel, EuiFlexGroup, EuiCommentList, EuiAccordion, EuiFlexItem } from '@elastic/eui';
+import React, { useEffect, useState, useMemo, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { ExceptionDetails } from './exception_details';
+import { ExceptionEntries } from './exception_entries';
+import { getFormattedEntries, getFormattedComments } from '../helpers';
+import { FormattedEntry, ExceptionListItemSchema } from '../types';
+
+const MyFlexItem = styled(EuiFlexItem)`
+  &.comments--show {
+    padding: 16px;
+    border-top: 1px solid #d3dae6;
```

Comment:
Are there no border colors from a constants of Eui we can utilize?

I see in a few spots there are things like so:

```ts
outline: solid 5px ${theme.eui.euiColorVis1_behindText};
```

Would be nice if we have a eui color constant we can use. Just pointing it out in case there is one out there. Otherwise this is fine for both day and night.

Reply:
Good call. There is indeed one for it. I updated this and other spots where I could use the theme properties. Thanks!